### PR TITLE
Fix: remove unused library dol_json

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

**Comment by hregis** [comment]
@eldy ping

File: htdocs/compta/facture/class/facture.class.php [hunk 1]
```diff
@@ -946,7 +946,7 @@ function createFromOrder($object)
 		$this->date = dol_now();
 		$this->source = 0;
 
-		$num=count($object->lines);
+		$num = (is_null($object->lines) ? 0 : count($object->lines)); // for avoid phpunit error : "Parameter must be an array or an object that implements Countable"
```
Comment (eldy) [inline htdocs/compta/facture/class/facture.class.php:949]: $object->lines should be an array. Which step do you do to get here a non array ?

Comment (hregis) [reply htdocs/compta/facture/class/facture.class.php:949]: @eldy 
I do not know, it was a mistake I had seen in Travis (php nightly build), you can remove this correction if you want.

**Comment by hregis** [comment]
I will redo this PR

**Pull request closed without merging**